presto-docs/src/main/sphinx/develop/system-access-control.rst

```diff
@@ -34,9 +34,8 @@ Configuration
 
 After a plugin that implements ``SystemAccessControl`` and
 ``SystemAccessControlFactory`` has been installed on the coordinator, it is
-configured using an ``etc/access-control.properties`` file. All of the properties
-other than ``access-control.name`` are specific to the ``SystemAccessControl``
-implementation.
+configured using a file specified in ``access-control.config-path`` property of ``config.properties`` (by default it points to ``etc/access-control.properties`` file).
```

Comment:
"specified in the ``access-control.config-path``"
"(by default, ``etc/access-control.properties``)"